src/components/views/elements/InterfaceScaleSlider.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 Marcel Radzio (MTRNord)
+Copyright 2017 Vector Creations Ltd.
```

Comment:
^^ this needs fixing

Reply:
What should this be changed to? This was a copy from another component. Should I be putting it under my own name?

Reply:
Yup, unless someone else wrote it :P

Reply:
ok 😄  and also remove `Copyright 2017 Vector Creations`?

Reply:
yup :)

Reply:
done 😃  hope my name is not taking too much space in the repo